components/scream/src/share/util/scream_common_physics_functions_impl.hpp

```diff
@@ -6,6 +6,24 @@
 
 namespace scream {
 
+template<typename DeviceT>
+template<typename ScalarT>
+KOKKOS_INLINE_FUNCTION
+ScalarT PhysicsFunctions<DeviceT>::calculate_dx_from_area(const ScalarT& area, const ScalarT& lat)
+{
+  using C = scream::physics::Constants<Real>;
+
+  static constexpr auto coeff_1 = C::earth_ellipsoid1;
+  static constexpr auto coeff_2 = C::earth_ellipsoid2;
+  static constexpr auto coeff_3 = C::earth_ellipsoid3;
+  static constexpr auto pi      = C::Pi; 
+
+  auto mpdeglat = coeff_1 - coeff_2 * std::cos(2.0*lat) + coeff_3 * std::cos(4.0*lat);
```

Comment:
Any chance we can use this opportunity to switch to a name that means something also to normal humans, instead of "mpdeglat"? :)

Reply:
Sure can... @bogensch , do you have an idea for a better variable name?

Reply:
First, I would vote to include the comments above this code block that the F90 version has, that should help make this variable name (and code block) descriptive.  If you desire a more verbose name in addition to this then m_per_degree_lat might be good.